index.html

```diff
@@ -98,6 +98,23 @@
         changes.
       </p>
 
+      <h2>
+        <a id="audit" class="anchor" href="#audit" aria-hidden="true"><span class="octicon octicon-link"></span></a>
+        Try Portus
+      </h2>
+
+      <p>
+        Portus is available on <a href="https://github.com/SUSE/Portus"> github</a>, but also as a ready to go <a href="https://github.com/SUSE/Portus/wiki/Installing-Portus"> appliance</a> based on openSUSE. <br>
```

Comment:
- github -> Github
- The appliance link should be https://github.com/SUSE/Portus/wiki/Installing-Portus#the-appliance